### PR TITLE
doc: add homebrew install instruction

### PR DESCRIPTION
**Pull request by chenrui333**
fixes:
- #43

---

relates to:
- https://github.com/Homebrew/homebrew-core/pull/80133

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +164,14 @@ COMMAND SUPPLIED
...
 `go get github.com/gruntwork-io/git-xargs@v0.0.5`
 
+
+### Installation option 3: Homebrew
+
+If you are [Homebrew](https://brew.sh/) user, you can install by running
+
+```bash
+$ brew install git-xargs
+```
```
Comment (brikis98) [inline README.md:174]: This is a much nicer install option, so would you mind making this option 1, at the top of the list?

**Review by brikis98 — COMMENTED** [review]
Thanks for adding Homebrew support and this PR!

File: README.md (hunk 1 above)
Comment (chenrui333) [reply README.md:174]: Oh, definitely. :)

Comment (chenrui333) [reply README.md]: Done with the update.

**Review by zackproser — APPROVED** [review]
LGTM! Thanks so much for getting the homebrew formula added and for writing up the docs!

**Comment by chenrui333** [comment]
my pleasure.

**Pull request merged**